<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ECCClass</class>
 <widget class="QMainWindow" name="ECCClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>517</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>517</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>517</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ECC</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="SK_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>私钥：</string>
    </property>
   </widget>
   <widget class="QLabel" name="PK_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>公钥：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SK_lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="PK_lineEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="inputFile_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>输入文件：</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputFile_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>输出文件：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputFile_lineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputFile_lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="input_toolButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="output_toolButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="encryption_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>加密</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decrypt_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>64</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>解密</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="signature_edit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>311</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="signature_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>签名</string>
    </property>
   </widget>
   <widget class="QPushButton" name="check_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>370</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>验证签名</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ECC.qrc"/>
 </resources>
 <connections/>
</ui>
